### PR TITLE
Add library names

### PR DESCRIPTION
**Pull request by lassik**

### CODE REVIEW

File: admin/srfi-data.scm [hunk 1]
```diff
@@ -1907,6 +1909,7 @@
  (status final)
  (title "Define higher-order lambda")
  (author "Lassi Kortela")
+ (library-name define)
```
Comment (arthurgleckler) [inline admin/srfi-data.scm:1912]: Way too broad.  How about `curried-define`?

File: admin/srfi-data.scm [hunk 2]
```diff
@@ -1873,6 +1874,7 @@
  (status final)
  (title "Central Log Exchange")
  (author "Göran Weinholt")
+ (library-name logging)
```
Comment (arthurgleckler) [inline admin/srfi-data.scm:1877]: How about `log-exchange`?

File: admin/srfi-data.scm [hunk 3]
```diff
@@ -1543,6 +1543,7 @@
  (status final)
  (title "Version flag")
  (author "Lassi Kortela")
+ (library-name version)
```
Comment (arthurgleckler) [inline admin/srfi-data.scm:1546]: This is too broad.  How about `version-flag`?

**Comment by lassik** [comment]
The above names are mentioned in the SRFI documents themselves. Except 176, which is from the Mosh Scheme implementation.

Each name must be paired with the SRFI number, so short names are better than long names. The names don't have to be unique by themselves. We have a few long names, and they are not obviously more descriptive than the short ones.

In the case of 219, it's named `define` because it exports the identifier `define`. Another SRFI could export another flavor of `define`. It makes sense to call both `define` since any given program is only likely to use one or the other.

**Comment by arthurgleckler** [comment]
You're right.  Sorry for the confusion.  I had forgotten that they would be combined with the SRFI number.

**Comment by lassik** [comment]
No problem. We should probably document this better somewhere, but I don't know where.

**Comment by lassik** [comment]
Well, they're a R6RS thing for now. We should start using them with R7RS as well.

**Pull request merged**